stream2hop/gcn2hop.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+import gcn.voeventclient
+from . import Utils as ut
+import sys
+import os
+from . import constant
+import argparse
+
+
+def _add_parser_args(parser):
+    """
+        adding parser arguments
+    """
+    ut.add_common_arguments(parser)
+
+    parser.add_argument(
+        "--hosts", default=constant.DHL, help="Comma separated list of GCN hosts."
+    )
+    parser.add_argument(
+        "-p", "--port", default=constant.PORT, type=int, help="GCN port"
+    )
+
+
+def _main(args=None):
+    """
+    Stream GCNs to Hop kafka server
+
+    """
+    if not args:
+        parser = argparse.ArgumentParser()
+        _add_parser_args(parser)
+        args = parser.parse_args()
+
+    #  Line buffer stdout and stderr
+    sys.stdout = os.fdopen(sys.stdout.fileno(), "w", buffering=1)
+    sys.stderr = os.fdopen(sys.stderr.fileno(), "w", buffering=1)
+
+    hopUrl = args.hop_url + "gcn"
```

Comment:
Use `urljoin` here as well.

Reply:
I tried to do so, but it didn't work with url starts with kafka://

Reply:
Oh right, my bad. That won't work here because it's a custom protocol.